bastion/docker-compose.yml

```diff
@@ -1,16 +1,36 @@
-version: '3.9'
+version: "3.9"
+name: vait
+
 services:
+  # Cloudflare tunnel to expose the services
+  cloudflared:
+    image: cloudflare/cloudflared:2024.2.1
+    restart: unless-stopped
+    command: tunnel run
+    environment:
+      - TUNNEL_TOKEN=$CF_TUNNEL_TOKEN
+  # CPU only Ollama instance for light model inference
   ollama:
-    image: ollama/ollama:0.1.27
+    container_name: vait-ollama-cpu
+    image: ollama/ollama:0.1.28
     restart: unless-stopped
-    networks:
-      - bridge
-    ports:
-      - 11434:11434
     volumes:
       - /opt/models:/root/.ollama
+  # LiteLLM Proxy to route LLM requests
+  litellm:
+    container_name: vait-litellm-proxy
+    image: ghcr.io/berriai/litellm:main-v1.29.1
+    restart: unless-stopped
+    ports:
+      - "8000:8000"
```

Comment:
I might need to double check. AFAIK containers in the same docker compose network can see each other ports :-/

LiteLLM Dockerfile is also exposing port 4000, should we use that instead of overriding to port 8000?

Reply:
Seems like the Dockerfile is wrong. It exposes port 4000 because the CMD section of Dockerfile specify the `--port 4000` option. Meanwhile the default port is 8000 per:
https://docs.litellm.ai/docs/proxy/cli#--port

The `--port 8000` option comes straight out of the vendor doc:
https://docs.litellm.ai/docs/proxy/quick_start

Given the default port is already at 8000, the command clause can be simplified to:
```
 command: "--config /app/config.yaml --num_workers $LITELLM_WORKERS_NUM"
 ```

```
vait-litellm-proxy  | INFO:     Started server process [1]
vait-litellm-proxy  | INFO:     Waiting for application startup.
vait-litellm-proxy  | INFO:     Application startup complete.
vait-litellm-proxy  | INFO:     Uvicorn running on http://0.0.0.0:8000 (Press CTRL+C to quit)
```

Reply:
> I might need to double check. AFAIK containers in the same docker compose network can see each other ports :-/

Yes they can. See the `config.yaml` and you see the litellm reference the `vait-ollama-cpu:11434`. However, we still need to expose port 8000 to ease the testing locally and provide quick endpoint monitoring from an external system.